calico-enterprise/getting-started/install-on-clusters/kubernetes/helm.mdx

```diff
@@ -38,7 +33,227 @@ In this guide, you install the Tigera Calico operator and custom resource defini
+  export EXT_SERVICE_PROTOCOL=<external-service-protocol>
+  ```
+  
+  If the external service is of type NodePort.
+
+  ```bash
+  export EXT_SERVICE_TYPE=<external-service-node-port>
+  ```
+
+1. Export one or more managed clusters.
+
+  ```bash
+  export MANAGED_CLUSTER_NAME=<managed-cluster-name>
+  export MANAGED_CLUSTER_OPERATOR_NAMESPACE=<managed-cluster-operator-namespace>
+  export MANAGED_CLUSTER_CERTIFICATE=<managed-cluster-certificate>
+  ```
+
+1. Install the Tigera {{prodname}} operator and custom resource definitions using the Helm 3 chart, with a NodePort service:
+
+  ```bash
+  helm install {{prodnamedash}} projectcalico/tigera-operator --version {{releaseTitle}} -f values.yaml \
+  --set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \
+  --set-file licenseKeyContent=<path/to/license/file/yaml> \
+
+  --set managementCluster.enabled=true \
+  --set managementCluster.address=$MANAGEMENT_CLUSTER_ADDR \
+  --set managementCluster.service.type=$EXT_SERVICE_TYPE \
+  --set managementCluster.service.port=$EXT_SERVICE_PORT \
+  --set managementCluster.service.targetPort=$EXT_SERVICE_TARGET_PORT \
+  --set managementCluster.service.protocol=$EXT_SERVICE_PROTOCOL \
+  --set managementCluster.service.nodePort=$EXT_SERVICE_NODE_PORT \
+  
+  --set managedClusters.enabled=true \
+  --set managedClusters.clusters[0].name=$MANAGED_CLUSTER_NAME \
+  --set managedClusters.clusters[0].operatorNamespace=$MANAGED_CLUSTER_OPERATOR_NAMESPACE \
+  --set managedClusters.clusters[0].certificate=$MANAGED_CLUSTER_CERTIFICATE \
+
+  --namespace tigera-operator --create-namespace
+  ```
+
+  or with a LoadBalancer on AWS, add expected annotations to the above installation.
+
+  ```bash
+  --set managementCluster.service.annotations[0].key="service.beta.kubernetes.io/aws-load-balancer-type" \
+  --set managementCluster.service.annotations[0].value=external \
+  --set managementCluster.service.annotations[1].key="service.beta.kubernetes.io/aws-load-balancer-nlb-target-type" \
+  --set managementCluster.service.annotations[1].value=instance \
+  --set managementCluster.service.annotations[2].key="service.beta.kubernetes.io/aws-load-balancer-scheme" \
+  --set managementCluster.service.annotations[2].value=internet-facing \
+  ```
+
+  :::note
+
+  The licenseKey must have `ApiVersion: crd.projectcalico.org/v1`.
+
+  :::
+
+1. You can now monitor progress with the following command:
+
+  ```bash
+  watch kubectl get tigerastatus
+  ```
+
+  Congratulations! You have now installed {{prodname}} for a management cluster using the Helm 3 chart.
+
+</TabItem>
+<TabItem label="Managed" value="Managed">
+
+To install a {{prodname}} [managed](../../../multicluster/create-a-managed-cluster#value) cluster with Helm:
+
+1. [Configure a storage class for Calico Enterprise](../../../operations/logstorage/create-storage).
```

Comment:
Managed clusters do not need a storage class.

Reply:
Removed